src/index.html

```diff
@@ -2,17 +2,78 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
     <title>Product cards</title>
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="/src/images/imac.jpeg"
+        alt="imac-logo"
+      />
+
+      <div class="card__details">
+        <span class="card__details-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </span>
+
+        <span class="card__details-identifier">Product code: 195434</span>
+      </div>
```

Comment:
Use semantic tags where possible instead of just divs and spans

Reply:
Whats better to use? `ul `and `li` tags ?

Reply:
I've changed all spans for semantic tags but I use all divs as a wrapper for moving blocks with content.